yarn-project/p2p/src/mem_pools/attestation_pool/memory_attestation_pool.test.ts

```diff
@@ -54,9 +54,30 @@ describe('MemoryAttestationPool', () => {
     expect(retreivedAttestationsAfterDelete.length).toBe(0);
   });
 
+  it('Should handle duplicate proposals in a slot', async () => {
```

Comment:
Added this test as I noticed two attestations coming in from the same node, however it ended up being that two were running on the same private key in the e2e tests